src/index.ts

```diff
@@ -57,6 +59,7 @@ ${chalk.bold('Options:')}
     --clean             Clear out the destination directory before install.
     --optimize          Transpile, minify, and optimize installed dependencies for production.
     --babel             Transpile installed dependencies. Also enabled with "--optimize".
+    --entry             Auto-detect imports from file(s). Supports glob.
```

Comment:
Last question: since we rely on glob now, and it’s not really an “entry”, what about renaming this to `--include` (e.g. `--include 'src/**/*.js'`)? (it’d also allow for a more intuitive `--exclude` followup, that I’d like to ship separately)

Reply:
+1, I'd prefer `--include` as well given the new behavior